base/client.jl

```diff
@@ -223,6 +223,11 @@ function exec_options(opts)
     global have_color     = (opts.color == 1) # --color=on
     global is_interactive = (opts.isinteractive != 0)
 
+    #check if stdout is TTY and color is not set manually
+    if isa(stdout, Base.TTY) && color_set == false
```

Comment:
```suggestion
    if isa(stdout, TTY) && !color_set
```
There's no need to qualify bindings inside of a module with the module's name. Also `x == false` is typically written `!x`.